src/main/java/org/terasology/projsndwv/commands/BeeCommands.java

```diff
@@ -146,17 +158,27 @@ public String birth(@Sender EntityRef client, @CommandParam(value = "count", req
         }
 
         EntityRef item = client.getComponent(ClientComponent.class).character.getComponent(CharacterHeldItemComponent.class).selectedItem;
-        if (item.getComponent(BeeComponent.class) == null || item.getComponent(MatedComponent.class) == null) {
-            return "Held item is not a mated bee.";
+        if (item.getComponent(BeeComponent.class) == null || item.getComponent(MatedComponent.class) == null || item.getComponent(BeeComponent.class).type != 2) {
```

Comment:
Prefer either
```java
BeeComponent component = item.getComoponent(BeeComponent.class);
if (component == null) {
    // Stuff
}
```
Or
```java
if (!item.hasComponent(BeeComponent.class) {
  //Stuff
}
```

Obviously whichever fits your style and code best but `hasComponent` is there for a reason